action.yml

```
@@ -87,14 +87,20 @@ runs:
   steps:
     - name: Install action dependencies
       shell: bash
-      run: python3 -m pip install clang-tools==0.6.2 cpp-linter==1.4.8
-    - name: Install clang-tools binary executables
-      shell: bash
-      run: clang-tools -i ${{ inputs.version }} -b
+      run: |
+        if [[ "${{runner.os}}"  == "macOS" ]];then
+          python3 -m venv venv
+          source venv/bin/activate
+        fi
+        python3 -m pip install -r ${{ github.action_path }}/requirements.txt
```

I found that it helps to quote the `action_path` var so the `\` on windows are treated as delimiters instead of escape sequences in bash.
```suggestion
        python3 -m pip install -r '${{ github.action_path }}/requirements.txt'
```